contents/uix/accessibility.md

```diff
@@ -0,0 +1,176 @@
+
+> Accessibility is the practice of making your websites usable by as many people as possible — we traditionally think of this as being about people with disabilities, but really it also benefits other groups such as those using mobile devices, or those with slow network connections. <sub>--[MDN Web Docs](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility)</sub>
+
+## Why is accessibility important?
+
+##### For the user
+Accessibility is important for people with disabilities to access the Web. There are many kinds of disabilities, including:
+   * Visual %%e.g., partial blindness, full blindness, color blindness, cataract, glaucoma, etc.%%
+   * Auditory %%e.g., hard of hearing, deafness, etc.%%
+   * Cognitive %%e.g., ADHD, autism spectrum, etc.%%
+   * Mobility %%e.g., quadriplegia, muscular dystrophy, etc.%%
+
+At the same time, accessibility also benefits people _without_ disabilities, for example:
+   * Device limitations %%e.g., using mobile phones, smart watches, smart TVs, and other devices with small screens, different input modes, etc.%%
+   * Age-related limitations %%e.g., poor dexterity, poor eyesight%%
+   * Temporary limitations %%e.g., broken arm, lost glasses%%
+   * Situational limitations %%e.g., under bright sunlight, an environment where one cannot listen to audio%%
+   * Internet-related limitations %%e.g., slow Internet connection, limited bandwidth%%
+
+##### For the developer
+1. **Accessibility helps you improve your website's UX**. [W3C](https://www.w3.org/standards/webdesign/accessibility) notes that accessibility practices are good design practices in general, which improve the UX of your website.
+2. **Accessibility is a right**. Caring about accessibility demonstrates good ethics/morals.
+3. **Accessibility makes your website more discoverable**. Web pages with good accessibility practices are ranked higher by search engine crawlers, making your website more discoverable and marketable to new users.
+4. **Accessibility helps you increase your user base**. According to the [World Health Organization world report on disability](https://www.who.int/disabilities/world_report/2011/report/en/), about 15% of the world's population live with some form of disability. That makes up more than 1 billion people. It is a significant population of users, so improving accessibility can help you reach more people.
+5. **Accessibility is part of the law in some places**. In [some countries](https://www.w3.org/WAI/policies/), abiding by accessibility guidelines is mandatory. Be careful not to break the law!
+
+## How can I improve the accessibility of my website?
+
+There are many ways to improve the accessibility of your website. Below are some tips, based on the type of disabilities they address.
+
+### Visual
+Users with visual impairments rely on assistive technologies such as a magnifier or a screen reader.
+
+##### Best practice: Use more visual indicators to convey a message.
```

Comment:
You can use an icon here to make the heading stand out, as this heading is supposed to have a special significance.
```
##### :fas-thumbs-up: Best practice: Use more visual indicators to convey a message
```
No need for full stop at the end?